### PR TITLE
Implements suggestions on introduction episode

### LINKED ISSUE #49: Introduction - suggested changes
Questions
What are the FAIR principles?
Why should I care to be FAIR?
- Add - How do I get started?

Objectives:
Identify the FAIR principles 
Recognize the importance of moving towards FAIR in research
Relate the components of this lesson to the FAIR principles

What is FAIR?
- Introduction to the FAIR principles (origin)
- What is meant with Findable, Accessible, Interoperable and Re-usable
- What are the FAIR principles NOT

Activity - poll or world cloud or discussion. Read the summary table of the 15 FAIR principles in Wilkinson et al. and ask the participants - After reading this:
What are the terms that you are most unfamiliar with? or are there any terms that are unknown for you?
Do you know where would you get started making your data FAIR?
Is there a principle that you find really easy to implement in your workflows?

Why FAIR?
- Re-usability - find examples of re-usability of climate data
- Reproducibility - not remarked enough yet - selfish reasons to make research reproducible - take some from: https://doi.org/10.1186/s13059-015-0850-7

How do I get started?
- To make the FAIR principles adaptable to your workflows, you can extract main elements: documentation, metadata, access to data, file formats, persistent identifiers, licences
- Tell them how these elements relate to FAIR

If time allows - Activity - asses one of the datasets in your project against the main elements - fill in this check list
- If somebody get this data set from you, would be able to understand the structure and content without asking you?
- Do you know who has access to this data set? Could somebody easily have access to this data set? How?
- Does this data set needs a proprietary software to be used?
- Does this data set has a persistent identifier or usage licence?

### PR DESCRIPTION
**Pull request by Peter9192**
This PR will implement all the suggestions from https://github.com/escience-academy/Lesson-FAIR-Data-Climate/issues/49

closes #49

### CODE REVIEW

File: _episodes/introduction.md [hunk 1]
```diff
@@ -5,43 +5,74 @@ exercises: 10
...
-> ## FAIR is not open
+> ## Discuss the different principles
 >
-> Some data cannot be shared openly, for example, because of privacy considerations.
-> For such situations, the EU H2020 Program Guidelines on FAIR Data say that the data should be *"As open as possible, as closed as necessary."*
+> Read the summary table of the 15 FAIR principles in [Wilkinson *et
+> al.*](https://doi.org/10.1038/sdata.2016.18) (See "Box 2: The FAIR Guiding
+> Principles"). After reading this, answer the following questions:
 >
+> - Are there any terms that you are unfamiliar with
```
Comment (rogerkuou) [inline _episodes/introduction.md]:
```suggestion
> - Are there any terms that you are unfamiliar with?
```

File: _episodes/introduction.md [hunk 2]
```diff
@@ -5,43 +5,74 @@ exercises: 10
...
+- Interoperable: the data can easily be used by machines or in data analysis workflows.
+- Re-usable: the data can easily be used by others for new research
+
+The FAIR principles have a strong focus on "machine-actionability". This means
+that the data should be easily readable by computers (and not only by humans).
+This is particularly relevant for working with and discovering new data.
+
+> ## What the FAIR principles are **not**
+>
+> - A standard: The FAIR principles need to be adapted and followed as much as
```
Comment (rogerkuou) [inline _episodes/introduction.md]:
```suggestion
> - A standard: The FAIR principles need to be adopted and followed as much as
```

File: _episodes/introduction.md [hunk 3]
```diff
@@ -58,50 +89,56 @@ CC-BY licence. Source: <https://doi.org/10.5281/zenodo.3695300>*
...
-> ## Select your case study for the tutorial
->
-> In the tutorial we will be using real use cases to assess the FAIRness of a
-> dataset and identify possibilities for improvement. Take a few minutes to find
-> a suitable use cases. This may be one of your own projects or (recent) papers,
-> or you may select another study you find interesting.
+For the domain of climate sciences, some standards have already been developed
+that you can use right away. In fact, you might already be using some of them
+without realizing it. NetCDF files, for example, already implement some of the
+FAIR principles around data modelling. But sometimes you need to find your own
```
Comment (rogerkuou) [inline _episodes/introduction.md]:
```suggestion
FAIR principles around data modeling. But sometimes you need to find your own
```

File: _episodes/introduction.md [hunk 4]
```diff
@@ -58,50 +89,56 @@ CC-BY licence. Source: <https://doi.org/10.5281/zenodo.3695300>*
...
+
+This tutorial contains six thematic episodes that cover the basic steps to
+making data FAIR. We will explain how these topics relate to the FAIR
+principles, what you can already do today, and where appropriate, we will point
+out existing standards for climate data.
+
+The episodes are:
+
+- Documentation: helps to make data reusable, especially for humans
+- Metadata: similar to documentation, although typically more structure and standardized
```
Comment (rogerkuou) [inline _episodes/introduction.md]:
```suggestion
- Metadata: similar to documentation, although typically more structured and standardized
```

File: _episodes/introduction.md [hunk 5]
```diff
@@ -58,50 +89,56 @@ CC-BY licence. Source: <https://doi.org/10.5281/zenodo.3695300>*
...
-> - Any input dataset(s) that were used for this study
-> - The methods, tools or models that were applied to these data
-> - A short description of the output data that was generated
-> - Additional information that you find relevant (e.g. the scientific domain,
->   project, journal, community)
+> - If somebody gets this dataset from you, would they be able to understand
+>   the structure and content without asking you?
+> - Do you know who has access to this dataset? Could somebody easily have access
+>   to this dataset? How?
+> - Does this dataset needs a proprietary software to be used?
```
Comment (rogerkuou) [inline _episodes/introduction.md]:
```suggestion
> - Does this dataset needs proprietary software to be used?
```

**Review by rogerkuou — APPROVED** [review]
@Peter9192 Hi Peter, I reviewed it. In general it looks really nice. I just used Grammarly to fix some minor things.

**Comment by Peter9192** [comment]
Thanks @rogerkuou ! I implemented your suggestions and I'll continue with merging then 👍

**Pull request merged**